tests/io-tests.dx

```diff
@@ -0,0 +1,20 @@
+
+include "io.dx"
+
+:p unsafeIO \().
+  withTempFile \fname.
+    withFile fname WriteMode \stream.
+      fwrite stream $ AsList _ "lorem ipsum\n"
+      fwrite stream $ AsList _ "dolor sit amet\n"
+    readFile fname
+> (AsList 27 "lorem ipsum
+> dolor sit amet
+> ")
+
+
+:p unsafeIO \().
+  withAlloc 4 \ptr:(Ptr Int).
+    for i:(Fin 4). store (ptr +>> ordinal i) (ordinal i)
```

Comment:
I'm not sure if we should be hijacking the nice `+>>` operator for a low-level operation that few people will be playing with.

Reply:
It's quite convenient. But we can always tack some more `>`s on the end if the real estate becomes too valuable.